recipes/loos/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -x
+# This install script is intended for conda-forge, and assumes the conda env
+# is already set up. If you need to set up a conda env, I suggest running
+# conda_build.sh instead.
+set -x
+
+#if [ "$platform" = "Linux" ]; then
+#    export CXX=`which g++`
+#elif [ "$platform" = 'Darwin' ]; then
+#    export CXX=`which clang++`
+#else
+#    echo "Unknown platform $platform, assuming g++"
+#    export CXX=`which g++`
+#fi
+echo "Begin loos build.sh"
+export CONDA_PREFIX=$BUILD_PREFIX
+echo "CXX =  $CXX "
+echo "CC =  $CC "
+
+scons PREFIX=$CONDA_PREFIX
+scons PREFIX=$CONDA_PREFIX install
```

Comment:
Why is this not `PREFIX=$PREFIX`?

Reply:
Internally, we use CONDA_PREFIX and PREFIX differently, so for my cognitive benefit I was doing it this way -- the setup is *supposed* to support building in a conda env and installing elsewhere, if you want to (not a relevant behavior here). 

When I looked through the log, PREFIX was set to this weird thing with a bunch of "placeholder" stuff in it, while BUILD_PREFIX was set to what CONDA_PREFIX is set to in a regular conda build. Internally, my scons scripts use the presence of CONDA_PREFIX as the indicator that we're in a conda env, which is why I have the `export CONDA_PREFIX=$BUILD_PREFIX ` line in build.sh

Really, all that's doing is controlling where `scons install` will put things -- if I'm going to be manually copying them anyway, it probably doesn't matter.

Reply:
BUILD_PREFIX is where the compilers are. PREFIX is where you should install the libraries and that's where python is.

Reply:
Ok, I'll make that change.